src/Validator.js

```diff
@@ -0,0 +1,31 @@
+const REGEX = {
+  CAR_NAME: /^[a-zA-Zㄱ-ㅎㅏ-ㅣ가-힣0-9]{1,5}$/,
+  TRY_CNT: /^[1-9][0-9]*$/,
+};
+
+const ERROR_MESSAGE = {
+  INVALID_NAME: '5자 이하 영어,한글,숫자의 조합을 입력하세요',
+  DUPLICATE_NAME: '중복된 이름의 자동차가 있습니다',
+  INVALID_NUMBER: '1이상의 양의 정수를 입력하세요',
+};
+
+const Validator = {
+  validateName: (names) => {
+    names.forEach((name) => {
+      if (!REGEX.CAR_NAME.test(name)) {
+        throw new Error(ERROR_MESSAGE.INVALID_NAME);
+      }
+    });
```

Comment:
names 중 일부가 validation에 걸리는 경우 error를 throw하도록 잘 구현해 주셨네용! 동일한 동작을 위해 다른 어떤 Array 기능을 사용해 볼 수 있을지 찾아봐도 재밌을 것 같아요~

Reply:
every를 사용해서 수정해봤습니다!
```suggestion
    const isCarNameError = names.every((name) => REGEX.CAR_NAME.test(name));
    if (!isCarNameError) {
      throw new Error(ERROR_MESSAGE.INVALID_NAME);
    }
```

Reply:
evey를 사용해서 수정해보았습니다!
```suggestion
    const isCarNameError = names.every((name) => REGEX.CAR_NAME.test(name));

    if (!isCarNameError) {
      throw new Error(ERROR_MESSAGE.INVALID_NAME);
    }
```

Reply:
👍👍 추가로 객체 내장메서드에는  every외에 some도 존재하는데요. 두 차이점을 한 번 찾아보셔도 재밌을 것 같습니다~!